tests/test_fingerprints.py

```diff
@@ -0,0 +1,98 @@
+
+file_path = expanduser('~')
+protein = next(oddt.toolkit.readfile('mol2', file_path + '/Desktop/ODDT/dane/10gs_protein.mol2'))
+protein.protein = True
+
+ligand = list(oddt.toolkit.readfile('mol2', file_path + '/Desktop/ODDT/dane/10gs_ligand.mol2'))[0]
+
+InteractionFingerprint_result = np.loadtxt(file_path + '/miniconda3/PyPliF_res')
+
+class TestFingerprintMethods(unittest.TestCase):
+    def test_InteractionFingerprint(self):
+        """Interaction Fingerprint test """
+        sum_interactions = np.sum(InteractionFingerprint_result, axis=0)
+
+        self.assertEqual(InteractionFingerprint_result.shape[1], 7)
+        self.assertEqual(InteractionFingerprint_result.shape[0], len(protein.atom_dict['resid']))
+
+        cc = close_contacts(protein.atom_dict, ligand.atom_dict, 4)[0]
+        self.assertEqual(len(cc), sum_interactions[0])
+
+        aromatic = pi_stacking(protein, ligand)[0]
+        self.assertEqual(len(aromatic), sum_interactions[1])
+
+        aromatic_1 = pi_cation(protein, ligand)[0]
+        self.assertEqual(len(aromatic_1), sum_interactions[2])
+
+        _, donors, strict = hbond_acceptor_donor(ligand, protein)
+        self.assertEqual(len(donors[strict]), sum_interactions[3])
+
+        acceptors, _, strict1 = hbond_acceptor_donor(protein, ligand)
+        self.assertEqual(len(acceptors[strict1]), sum_interactions[4])
+
+        plus, _ = salt_bridge_plus_minus(protein, ligand)
+        self.assertEqual(len(plus), sum_interactions[5])
+
+        _, minus = salt_bridge_plus_minus(ligand, protein)
+        self.assertEqual(len(minus), sum_interactions[6])
+
+    def test_SimpleInteractionFingerprint(self):
+        """Simple Interaction Fingerprint test """
+        result = np.array([[0., 0., 0., 0., 0., 0., 0., 0.],
+                       [0., 0., 0., 1., 0., 0., 0., 0.],
+                       [0., 0., 0., 0., 0., 0., 0., 0.],
+                       [0., 0., 0., 0., 1., 0., 0., 0.],
+                       [0., 0., 0., 0., 0., 0., 0., 0.],
+                       [0., 0., 0., 0., 2., 0., 0., 0.],
+                       [0., 0., 0., 0., 0., 0., 0., 0.],
+                       [0., 0., 0., 0., 0., 0., 0., 0.],
+                       [0., 0., 0., 0., 0., 0., 0., 0.],
+                       [0., 0., 0., 0., 0., 0., 0., 0.],
+                       [0., 0., 0., 1., 1., 0., 0., 0.],
+                       [0., 0., 0., 1., 0., 0., 0., 0.],
+                       [0., 0., 0., 0., 0., 0., 0., 0.],
+                       [6., 1., 0., 0., 0., 0., 0., 0.],
+                       [0., 0., 0., 0., 0., 0., 0., 0.],
+                       [0., 0., 0., 2., 1., 0., 0., 0.],
+                       [0., 0., 0., 0., 0., 0., 0., 0.],
+                       [0., 1., 0., 1., 0., 0., 0., 0.],
+                       [0., 0., 0., 0., 0., 0., 0., 0.],
+                       [2., 0., 0., 0., 0., 0., 0., 0.],
+                       [0., 0., 0., 0., 0., 0., 0., 0.]])
+        sum_interactions = np.sum(result, axis=0)
```

Comment:
You are testing here interaction module, not interaction FPs. Just call the `InteractionFingerprint`, `SimpleInteractionFingerprint` and check with the array above using `assert_array_equal`. There are tests already for the interaction module elsewhere